src/main/java/com/users/validation/EmailListValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.users.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class EmailListValidator implements ConstraintValidator<ValidEmailList, List<String>> {
+
+    private static final String EMAIL_REGEX = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$";
```

Comment:
uite de ce zic sa ai o clasa de constante, ca ai acelasi regex in 3 clase

Reply:
am facut o clasa asa, gata